Expensify/App
# Disallow passing CompanyStep w/ empty phone

### Details

We could improve this later to also _validate_ the phone number beyond its existence e.g. make sure it is numbers etc. But for now we can just require the phone number in the front end vs. worry about handling the BE error (which throws but provides no feedback at the moment).

### Fixed Issues (Follow up)
$ https://github.com/Expensify/App/issues/4836

### Tests / QA Steps
1. Create a new workspace
1. Navigate to that workspace
1. Select Expensify Cards
1. Select “Get Started” button
1. Enter information for bank account
1. On company information step fill out all required fields
1. Verify that the phone number is also required and we cannot continue without entering something into this field.
![2021-09-01_13-21-51](https://user-images.githubusercontent.com/32969087/131759072-42315b92-8e3d-446b-8e3f-6ba4fdd49ab4.png)

### Tested On

- [x] Web - Skipping tests on other platforms because this is a minor change and I don't expect there to be any differences

### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
